docs/pages/signetjs/contracts/solana/get-derived-public-key.mdx

```diff
@@ -0,0 +1,26 @@
+## getDerivedPublicKey
+
+Derives a public key based on a predecessor address and a derivation path.
+
+### Usage
+
+```ts
+const publicKey = await solanaChainSigContract.getDerivedPublicKey({
```

Comment:
Is this done locally on the client side?

Reply:
Yes